### PR TITLE
HDDS-11927. Fix flaky TestContainerBalancerStatusInfo.testGetCurrentStatisticsWhileBalancingInProgress

### PR DESCRIPTION
**Pull request by juncevich**
## What changes were proposed in this pull request?
Fix flaky TestContainerBalancerStatusInfo.testGetCurrentStatisticsWhileBalancingInProgress

https://issues.apache.org/jira/browse/HDDS-11927

### CODE REVIEW

**Comment by adoroszlai** [comment]
Thanks @juncevich for the patch.  Would you mind running [`flaky-test-check`](https://github.com/juncevich/ozone/actions/workflows/intermittent-test-check.yml) for `TestContainerBalancerStatusInfo` to verify the fix?  (Please set `Test Name = ALL`, not just the flaky method.)

**Comment by juncevich** [comment]
> Thanks @juncevich for the patch.  Would you mind running [`flaky-test-check`](https://github.com/juncevich/ozone/actions/workflows/intermittent-test-check.yml) for `TestContainerBalancerStatusInfo` to verify the fix?  (Please set `Test Name = ALL`, not just the flaky method.)

Thank you. Will check problems.

File: hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/TestContainerBalancerStatusInfo.java [hunk 1]
```diff
@@ -162,9 +161,9 @@ void testGetCurrentStatisticsWhileBalancingInProgress() throws Exception {
 
     ContainerBalancerTask task = mockedScm.startBalancerTaskAsync(config, false);
     // Get the current iteration statistics when it has information about the containers moving.
-    LambdaTestUtils.await(1000, 10,
-        () -> task.getCurrentIterationsStatistic().size() == 2 &&
-              task.getCurrentIterationsStatistic().get(1).getContainerMovesScheduled() > 0);
+    LambdaTestUtils.await(5000, 1, () -> task.getCurrentIterationsStatistic().size() == 2 &&
+                                         task.getCurrentIterationsStatistic().get(1).getContainerMovesScheduled() > 0
```
Comment (adoroszlai) [inline hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/TestContainerBalancerStatusInfo.java:165]: nit: Let's keep previous formatting.

Comment (juncevich) [reply hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/TestContainerBalancerStatusInfo.java]: Done.

**Comment by juncevich** [comment]
@adoroszlai could you review this PR?

**Review by adoroszlai — APPROVED** [review]
Thanks @juncevich for updating the patch.

**Pull request merged**